src/unicore.cpp

```diff
@@ -179,5 +179,31 @@ bool UnicoreParser::extractHeading()
 
 bool UnicoreParser::extractAgrica()
 {
-	return true;
-}
+	// Skip to ;
+	char *ptr = strchr(_buffer, ';');
+
+	if (ptr == nullptr) {
+		return false;
+	}
+
+	ptr = strtok(ptr, ",");
+
+	unsigned i = 0;
+
+	while (ptr != nullptr) {
+		ptr = strtok(nullptr, ",");
+
+		if (ptr == nullptr) {
+			return false;
+		}
+
+		if (i == 24) {
```

Comment:
Where are you getting 24?

Reply:
I'm not sure if I have the right document.

![image](https://github.com/user-attachments/assets/4f9ffdc1-058b-49a2-aae1-cb88b3033e14)

Reply:
Including the speed accuracy is also super valuable.

Reply:
It's off by a few due to the header. Basically trial and error to get it right.

Ok, I will include speed accuracy.

Reply:
@dagar I've added speed accuracy, and moved all speed fields over to AGRICA.